### DISCUSSION TITLE
Cancel Application

### DISCUSSION THREAD

**Discussion by urbfbaby** [community/community#37279] [GitHub Education]
hello everyone, I'm confused how do I cancel the registration of the student developer pack, I used to register and was accepted but for some reason now it can't be accessed anymore, then I tried to re-register, then a message like the one below appears

**Your application cannot be reviewed until you fix the following:
Modify your browser settings to allow GitHub to use your location information. Once location is allowed you MUST cancel your current application and begin again.
You are significantly more likely to be verified if you have completed your [GitHub user profile] with your full name and a short bio.**

How to handle that? Thank you

**Comment by imqcro77** [comment]
Have you fixed this?

**Comment by urbfbaby** [comment]
Nope,

**Reply by samus-aran** [reply]
Hi @urbfbaby 👋 
Do you have a screenshot of the errors you're experiencing on the application page?

**Comment by WeaThox** [comment]
I have exactly the same problem...

Go to your browser's settings and autorize the acces of your location for the website github. After that refresh your page and retry.  

![Capture d’écran 2022-11-09 231838](https://user-images.githubusercontent.com/66534822/200954132-fa593eee-ae25-4208-b512-542da82f4fd7.jpg)

![Capture d’écran 2022-11-09 231932](https://user-images.githubusercontent.com/66534822/200954315-4ee3c838-9690-4249-9ee1-00a1379487c6.jpg)
